src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorPortal.class.php

```diff
@@ -457,10 +470,38 @@ protected function CopyNodeDivision($aSourceNode, $sTargetNodeId)
             unset($aDefaultData['id']);
             $aDefaultData['cms_portal_id'] = $this->sId;
             $aDefaultData['cms_tree_id_tree'] = $sTargetNodeId;
+            $oTableManager->ForceHiddenFieldWriteOnSave(true);
+            $oTableManager->Save($aDefaultData);
+        }
+    }
+
+    /**
+     * copies the navigations attached to a tree node to a new tree node
+     *
+     * @param array  $aSourceNode   - sqlData of the source node
+     * @param string $sTargetNodeId - id of the target tree node
+     */
+    protected function CopyNodeNavigations($aSourceNode, $sTargetNodeId)
+    {
+        // FIXME: This does not copy navigations that are not attached to the tree
+        $navigation = TdbCmsPortalNavigation::GetNewInstance();
+        if ($navigation->LoadFromField('tree_node', $aSourceNode['id'])) {
+            $navigationTableConf = $navigation->GetTableConf();
+
+            /** @var $oTableManager TCMSTableEditorManager */
+            $oTableManager = new TCMSTableEditorManager();
+            $oTableManager->Init($navigationTableConf->id, null);
+            $oTableManager->AllowEditByAll($this->bAllowEditByAll);
```

Comment:
IMHO we should just set `AllowEditByAll` to `true` here (or rather in the newer version of the this), because if you copy a portal you need to copy the navigations no matter what. I don't think that `Save` would fail, but it just makes more sense to always allow the edit here.

I don't know if @rnuernberg has a different opinion though.

Reply:
Just set true here. That's totally o.k.

Reply:
Alright. I set it to true.